<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Sample API" tests="1" time="0.000">
  <testsuite name="List all users" id="a06edcee-e4a8-4c90-b2a6-12f3b5fbec76" timestamp="2025-05-30T12:18:29.061Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: Error: connect ECONNREFUSED 127.0.0.1:3000
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1549:16)

---
]]></system-err>
    <testcase name="Status code is 2xx" time="0.000" classname="ListAllUsers">
      <failure type="AssertionFailure" message="expected PostmanResponse{ …(5) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 656641ee-a18b-4c3b-a310-9122bc0271eb.]]>
        <![CDATA[Collection name: Sample API.]]>
        <![CDATA[Request name: List all users.]]>
        <![CDATA[Test description: Status code is 2xx.]]>
        <![CDATA[Error message: expected PostmanResponse{ …(5) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected PostmanResponse{ …(5) } to have property 'code'
   at Object.eval sandbox-script.js:1:1).]]>
      </failure>
    </testcase>
  </testsuite>
</testsuites>